libs/src/clients/btc/rpc/ankr.py

```
@@ -209,3 +132,12 @@ async def get_latest_block_number(self) -> BlockNumber:
         }
         resp = await self._request("POST", url, headers=headers, json_data=data)
         return resp["result"]["blocks"]  # type: ignore
+
+    async def get_fee_per_byte(self) -> int | Decimal | None:
```

instead of returning None, is it not better to raise an error?